benchmarks/pure_benchmark.ml

```diff
@@ -13,16 +13,11 @@ let read file =
   with_file ~mode:[O_RDONLY] file ~f:(fun fd ->
     loop 0 size fd)
 
-let many' p =
-  let open Angstrom in
-  fix (fun m ->
-    (p *> m) <|> return ())
-
 (* For input files involving trailing numbers, .e.g, [http-requests.txt.100],
  * go into the [benchmarks/data] directory and use the [replicate] script to
  * generate the file, i.e.,
  *
- *   [./replicate http-requests.txt.100]
+ *   [./replicate http-requests.txt 100]
```

Comment:
The change was intended. First argument is file name, second is how many times it should appear in the output. The output file name is the concatenation of the two arguments, with a period in between.